src/AppBundle/Entity/JemarcheReport.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * JemarcheReport
+ *
+ * @ORM\Table(name="jemarche_report")
+ * @ORM\Entity()
+ */
+class JemarcheReport
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="actionType", type="string", length=50)
```

Comment:
Please remove all the columns names and all the column types if they are strings.